experimental/packages/opentelemetry-instrumentation-xml-http-request/src/xhr.ts

```diff
@@ -302,10 +339,15 @@ export class XMLHttpRequestInstrumentation extends InstrumentationBase<XMLHttpRe
         this._addChildSpan(span, corsPreFlightRequest);
         this._markResourceAsUsed(corsPreFlightRequest);
       }
+      const skipOldSemconvContentLengthAttrs = !(
```

Comment:
I approved this on fetch but now I'm wondering: this says to skip old if NOT `SemconvStability.OLD`, right? To keep existing behavior in place for http/dup, should this instead be skip old semconv if `SemconvStability.STABLE`?

Reply:
Nope, because `this._semconvStability & SemconvStability.STABLE` is `true` if using `http/dup`.

In other words, when using `http/dup`, `skipOldSemconvContentLengthAttrs` will be false, which keeps the old behaviour.

Reply:
This test assert tests this: https://github.com/trentm/opentelemetry-js/blob/trentm-instr-xhr-http-semconv-migration/experimental/packages/opentelemetry-instrumentation-xml-http-request/test/xhr.test.ts#L1845-L1852

Reply:
Got it, thanks!